app/questionnaire/rules.py

```diff
@@ -8,19 +8,15 @@ def evaluate_rule(when, answer_value):
     match_value = when['value']
     condition = when['condition']
 
-    if isinstance(answer_value, list):
-        if len(answer_value) == 1:
-            answer_to_test = answer_value[0]
-        else:
-            return False
-    else:
-        answer_to_test = str(answer_value)
+    answer_to_test = str(answer_value) if 'equals' in condition else answer_value
```

Comment:
Ah is this converting everything with 'equals' in the title to a string? If so, then you probably should be explicit about it:

     if condition in ['equals', 'not equals']